client/components/Application/Dialogs/WelcomeBanner.tsx

```diff
@@ -0,0 +1,90 @@
+import Box from '@mui/material/Box'
+import Dialog from '@mui/material/Dialog'
+import DialogContent from '@mui/material/DialogContent'
+import SimpleButton from '../../Parts/Buttons/SimpleButton'
+import Checkbox from '@mui/material/Checkbox'
+import * as store from '@client/store'
+import * as React from 'react'
+import welcomescreenImg from './assets/welcomescreen.png'
+
+export default function WelcomeBanner() {
+  const [checked, setChecked] = React.useState(false)
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setChecked(event.target.checked)
+    // TODO: how to save value directly to the store, do i have access to the store
+    // or do i need to write an action?
+    // store.setState({'hideWelcomeBanner', checked})
```

Comment:
hello @roll i got stuck here trying to do this change. With the new store rewrite, is it possible to write directly to the store? or do I need to create an action? if it's possible to write to the store without an action, what would be the new syntax? :)

Reply:
Hi @guergana, yes, you need to write an action